### PR TITLE
Remove all the times env() is called outside of the config files

### PR DESCRIPTION
**Pull request by ysbrandB**
With caching these will return null.
This has not been happening because the double routes prevented caching but will become a problem otherwise.

### CODE REVIEW

File: config/directadmin.php
Comment (MaxLiebe) [inline config/directadmin.php:1]: Might as well change these to env() instead of getenv() for consistency (and to use Laravel's syntaxes)

**Review by MaxLiebe — CHANGES_REQUESTED** [review]
LGTM, just do the smol change I requested and it can be merged

File: config/directadmin.php
Comment (MaxLiebe) [reply config/directadmin.php:1]: Also makes more sense as env() returns null if the variable doesn't exist whereas getenv() returns false

Comment (ysbrandB) [reply config/directadmin.php:1]: resolved

**Pull request merged**